.github/workflows/iroha2-dev.yml

```diff
@@ -19,10 +17,22 @@ jobs:
       image: 7272721/i2-ci:nightly
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
```

Comment:
This is a requirement from [`tj-actions/changed-files`](https://github.com/tj-actions/changed-files#usage) for `push` workflows